ynput/OpenPype
# Autosave preferences can be read after Nuke opens the script

## Changelog Description
Looks like I need to open the script in Nuke to be able to correctly load the autosave preferences.
This PR reads the Nuke script in context, and offers owerwriting the current script with autosaved one if autosave exists.

## Testing notes:
1. Launch Nuke via Openpype
2. Go to Openpype menu and click work file
3. Open any nuke script with autosave
4. It will pop up nuke prompt to ask if you want to open autosave script.
5. It will load autosave script for you if you click yes; while it wont if you click no.

Also try to change autosave location in Nuke prefs to some random place, wait till autosave kicks in and kill Nuke.
OP should detect and offer autosaves that is not located next to the workfile as well.
